examples/acquiring_data/goes_xrs_example.py

```diff
@@ -69,37 +66,6 @@
 goes_15 = ts.TimeSeries(file_goes15)
 goes_15.peek()
 
-#############################################################
-# The resulting `~sunpy.timeseries.TimeSeries` can be filtered by GOES quality flags. For more information
-# refer to the `GOES Data Guide <https://satdat.ngdc.noaa.gov/sem/goes/data/science/xrs/GOES_13-15_XRS_Science-Quality_Data_Readme.pdf>`__.
-
-df = goes_15.to_dataframe()
-df = df[(df["xrsa_quality"] == 0) & (df["xrsb_quality"] == 0)]
-goes_15 = ts.TimeSeries(df, goes_15.meta, goes_15.units)
```

Comment:
I wonder if this needs to stay but this becomes a tutorial or something? Or maybe text in the GOES timeseries class about the quality flags?

Reply:
👍 I can move this to a temporary folder for future tutorial material so it's not lost